packages/core/README.md

```diff
@@ -0,0 +1,30 @@
+# ESLint Core
+
+## Overview
+
+This package is the future home of the rewritten, runtime-agnostic ESLint core.
+
+Right now, it simply exports the core types necessary to implement language plugins.
+
+## License
+
+Apache 2.0
+
+## Sponsors
+
+The following companies, organizations, and individuals support ESLint's ongoing maintenance and development. [Become a Sponsor](https://eslint.org/donate) to get your logo on our README and website.
+
+<!-- NOTE: This section is autogenerated. Do not manually edit.-->
+<!--sponsorsstart-->
+<h3>Platinum Sponsors</h3>
+<p><a href="https://automattic.com"><img src="https://images.opencollective.com/automattic/d0ef3e1/logo.png" alt="Automattic" height="undefined"></a></p><h3>Gold Sponsors</h3>
+<p><a href="#"><img src="https://images.opencollective.com/guest-bf377e88/avatar.png" alt="Eli Schleifer" height="96"></a> <a href="https://engineering.salesforce.com"><img src="https://images.opencollective.com/salesforce/ca8f997/logo.png" alt="Salesforce" height="96"></a> <a href="https://www.airbnb.com/"><img src="https://images.opencollective.com/airbnb/d327d66/logo.png" alt="Airbnb" height="96"></a></p><h3>Silver Sponsors</h3>
```

Comment:
height is undefined? 🤔

Reply:
That's how the script works, probably because there's no predefined fixed height for Platinum sponsors:

https://github.com/eslint/rewrite/blob/85018903815908bcfd940bbda347f3f17663e4bd/tools/update-readme.js#L35-L39

I guess it should be added, or the script should be updated to omit the height attribute.

Reply:
Yeah, we should fix the script.

Also WOW! Airbnb is now a platinum sponsor????

Reply:
That's the info received from OC few days ago: https://github.com/eslint/eslint.org/commit/1741c34fa63a5ba86d9ba8c52a3f9da2f87150c4